### DISCUSSION TITLE
CAPTCHAs

### DISCUSSION THREAD

**Discussion by ZeaMays14142** [Integration-Automation/ReEdgeGPT#48] [General]
I’ve recently encountered an issue with CAPTCHAs while testing reedgegpt. Specifically, I ran into an error in my script due to encountering a CAPTCHA when I tried to run a test of reedgegpt to remotely get a response to bingAI.

As a workaround, I tried adding a delay between using the cookie to log in and asking the question. However, I’m not sure if this is the most effective solution and I’m interested in exploring other strategies.

I’m wondering if it might be better to copy all cookies and not just the MUID login one. I’m also curious about other ways to work around CAPTCHAs and would like to hear more about your CAPTCHA handling strategies.

**Comment by JE-Chen** [comment]
Copy all cookies -> yes, use Cookie-Editor to export all cookie to Json like what README said.

CAPTCHAs: 
* Open your browser like Edge ([Or other browser like Firefox and set User-Agent to Edge](https://github.com/tom-snow/PyroEdgeGPTBot#checking-bing-ai-access-required))
* Login into [bing.com/chat](https://www.bing.com/chat) (login may not required, you can skip login and do steps below)
* Ask a question and you will face the CaptchaChallenge.
* Finish the CaptchaChallenge, then Bing AI works for you.
* Export the Cookie now. (This time the cookie have more keys, these keys may pass the CaptchaChallenge)

**Reply by ZeaMays14142** [reply]
ok, thanks.
im now trying this, but im having ssl handshake errors, so the cookies dont even get saved (which they did a week ago)

**Reply by JE-Chen** [reply]
Well, I think  SSL handshake errors not this package problem?

**Reply by ZeaMays14142** [reply]
yes, they arent
